src/screens/ProposalDetail.js

```diff
@@ -25,12 +26,18 @@ import usePanelState from '../hooks/usePanelState'
 import { useWallet } from '../providers/Wallet'
 
 import { getTokenIconBySymbol, formatTokenAmount } from '../lib/token-utils'
-import { addressesEqualNoSum as addressesEqual } from '../lib/web3-utils'
+import {
+  addressesEqualNoSum as addressesEqual,
+  soliditySha3,
+} from '../lib/web3-utils'
+
+const cancelRoleHash = soliditySha3('CANCEL_PROPOSAL_ROLE')
```

Comment:
Let's rename it to `CANCEL_PROPOSAL_ROLE` as it is in the contract?

Reply:
here i would prefer to make explicit with the name that is the hash and not the string so just reading the name you know what is it :)

Reply:
Ok that's fair. Maybe have it in upper case though as we do with all constants?